SWLOR.Game.Server/Feature/ShipModuleDefinition/ShipConfigurationModuleDefinition.cs

```diff
@@ -0,0 +1,142 @@
+            ShipConfig("con_int6", "NPC Int Conf 6", "iit_ess8_121", 1, 0, 0, 0, 125, 0, 160, 0, 240, 30, 30, 0);
+
+            ShipConfig("con_bmb1", "NPC Bmb Conf 1", "iit_ess8_121", 1, 0, 0, 0, 35, 0, 35, 5, 25, 2, 2, 0);
+            ShipConfig("con_bmb2", "NPC Bmb Conf 2", "iit_ess8_121", 1, 0, 0, 0, 50, 0, 50, 10, 38, 5, 2, 0);
+            ShipConfig("con_bmb3", "NPC Bmb Conf 3", "iit_ess8_121", 1, 0, 0, 0, 70, 0, 70, 15, 50, 10, 2, 0);
+            ShipConfig("con_bmb4", "NPC Bmb Conf 4", "iit_ess8_121", 1, 0, 0, 0, 90, 0, 90, 20, 70, 20, 2, 0);
+            ShipConfig("con_bmb5", "NPC Bmb Conf 5", "iit_ess8_121", 1, 0, 0, 0, 120, 0, 120, 25, 90, 30, 30, 0);
+            ShipConfig("con_bmb6", "NPC Bmb Conf 6", "iit_ess8_121", 1, 0, 0, 0, 160, 0, 160, 30, 125, 30, 30, 0);
+
+            ShipConfig("con_hvy1", "NPC Hvy Conf 1", "iit_ess8_121", 1, 0, 0, 0, 50, 0, 35, 0, 25, 2, 2, 0);
+            ShipConfig("con_hvy2", "NPC Hvy Conf 2", "iit_ess8_121", 1, 0, 0, 0, 75, 0, 50, 0, 38, 5, 2, 0);
+            ShipConfig("con_hvy3", "NPC Hvy Conf 3", "iit_ess8_121", 1, 0, 0, 0, 115, 0, 70, 0, 50, 10, 2, 0);
+            ShipConfig("con_hvy4", "NPC Hvy Conf 4", "iit_ess8_121", 1, 0, 0, 0, 145, 0, 90, 0, 70, 20, 2, 0);
+            ShipConfig("con_hvy5", "NPC Hvy Conf 5", "iit_ess8_121", 1, 0, 0, 0, 180, 0, 120, 0, 90, 30, 30, 0);
+            ShipConfig("con_hvy6", "NPC Hvy Conf 6", "iit_ess8_121", 1, 0, 0, 0, 240, 0, 160, 0, 125, 30, 30, 0);
+
+            ShipConfig("con_cap1", "NPC Cap Conf 1", "iit_ess8_121", 1, 0, 0, 30, 250, 30, 175, 30, 125, 30, 0, 0);
+            ShipConfig("con_cap2", "NPC Cap Conf 2", "iit_ess8_121", 1, 0, 0, 30, 275, 30, 190, 30, 140, 30, 0, 0);
+            ShipConfig("con_cap3", "NPC Cap Conf 3", "iit_ess8_121", 1, 0, 0, 30, 300, 30, 205, 30, 155, 30, 0, 0);
+            ShipConfig("con_cap4", "NPC Cap Conf 4", "iit_ess8_121", 1, 0, 0, 30, 325, 30, 220, 30, 170, 30, 0, 0);
+            ShipConfig("con_cap5", "NPC Cap Conf 5", "iit_ess8_121", 1, 0, 0, 30, 350, 30, 235, 30, 195, 30, 0, 0);
+            ShipConfig("con_cap6", "NPC Cap Conf 6", "iit_ess8_121", 1, 0, 0, 30, 375, 30, 250, 30, 210, 30, 0, 0);
+            ShipConfig("con_cap7", "NPC Cap Conf 7", "iit_ess8_121", 1, 0, 0, 30, 400, 30, 275, 30, 225, 30, 0, 0);
+
+            return _builder.Build();
+        }
+
+        private void ShipConfig(string itemTag, string name, string texture, int requiredLevel, int armor, int shield, int thermalAttack, int thermalDefense, int ionAttack, int ionDefense, int explosiveAttack, int explosiveDefense, int accuracy, int evasion, int industrial)
+        {
+            _builder.Create(itemTag)
+                .Name(name)
+                .ShortName(name)
+                .Texture(texture)
+                .Description($"Provides several base stats to a ship to allow it to fulfill a role and operate at full capacity: \n" +
+                $"Armor: +{armor} + Module Bonus * 4 \n" +
```

Comment:
Since this is being displayed to players, I'd recommend using "x4" instead of "* 4" simply because non-programmers might not understand what that notation means.